studio/hooks/analytics/useDbQuery.tsx

```diff
@@ -27,6 +27,8 @@ const useDbQuery = (
 
   const resolvedSql = typeof sql === 'function' ? sql([]) : sql
 
+  params.sql = resolvedSql
```

Comment:
Probably not 100% correct, but `params` was missing the SQL. Only worked for useLogsQuery but not for useDbQuery.